### ISSUE TITLE
Js/BrowserViewportWindow: Multiple event listeners are added whenever the window is resized

### ISSUE THREAD

**Issue by OliverO2** [OliverO2/compose-counting-grid#3]
From a private conversation in Slack:
> Have you ever typed on a web app developed in Compose web (canvas)? I recently noticed that multiple event listeners are added whenever I resize the window.

Check out this issue: https://github.com/BKMbigo/fund-a-school/issues/1

**Comment by OliverO2** [comment]
`SkiaLayer.attachTo` ignores its parameter `autoDetach: Boolean = true` and will indeed attach event listeners each time it is called.

Seemingly, this does not always lead to keystrokes being repeated, as described in https://github.com/BKMbigo/fund-a-school/issues/1 . For example, when changing `main()` as follows, keys will still work as ususal.
```kotlin
fun main() {
    onWasmReady {
        BrowserViewportWindow("Compose Counting Grid") {
            Column {
                MainScene()
                var text by remember { mutableStateOf("") }
                TextField(value = text, onValueChange = { text = it })
            }
        }
    }
}
```

Unfortunately, the workaround in https://github.com/BKMbigo/fund-a-school/issues/1 will not work with Kotlin 1.9.0. At compile time, accessing `layer.layer.state` will produce the warning
```
Property accessor 'state.<get-state>' can not be called: Private property accessor declared in module <org.jetbrains.skiko:skiko> can not be accessed in module <com.example:compose-counting-grid>
```
At run time, resizing will then crash with
```
IrLinkageError: Property accessor 'state.<get-state>' can not be called: Private property accessor declared in module <org.jetbrains.skiko:skiko> can not be accessed in module <com.example:compose-counting-grid>
```

**Comment by OliverO2** [comment]
The above commit should fix this issue with Kotlin 1.9.0 and Compose Multiplatform 1.5.0-beta01.
